### ISSUE TITLE
pnpm + svite = borked shell

### ISSUE THREAD

**Issue by codechips** [dominikg/svite#57]
## Description

Currently the combo pnpm + svite borks the shell when stopped (ctrl-c). Can't use autocomplete on files, etc. You have to press ctrl-c twice to clear everything up properly. I am thinking it's some kind of weird or missing escape sequence, but don't know if it's in vite, svite or pnpm. I am on Linux using Alacritty and zsh with oh-my-zsh.

## Reproduction

Create a simple svite app with pnpm as manager. Start the app and then stop it with Ctrl-C. Try to do ls and tab for autocomplete. That does not work for me unless I press ctrl-c again.

## Additional Information

- OS: Linux pop-os 5.4.0-7642-generic #46~1598628707~20.04~040157c-Ubuntu 
- pnpm 5.5.12

**Comment by dominikg** [comment]
Can you share the exact commands used to create the app (most importantly the template) and the output of the command you Ctrl-C'd? Preferably start it with --debug flag for svite please. Thanks.

Suspecting routify-mdsvex and the issue being caused by run-p not propagating the signal.

**Comment by codechips** [comment]
## Bootstrap

```
$ npx svite create -pm pnpm -ts -t postcss-tailwind -sc borked-shell
```
## Logs

```
$ pnpx svite -ts --debug
 --debug
  svite: setting process.env.NODE_ENV=development +0ms
  vite:config env mode: development +0ms
  vite:config env: {} +0ms
  svite: adding svite plugin to vite +168ms
  svite: added "svelte" to list of fields to resolve in package.json [ 'svelte', 'module', 'jsnext', 'jsnext:main', 'browser', 'main' ] +43ms
  svite: disabling vue support in vite +1ms
  svite: added svelte to rollupDedupe svelte/animate, svelte/easing, svelte/internal, svelte/motion, svelte/store, svelte/transition, svelte +0ms
  svite: vite config {
  env: {},
  debug: true,
  root: undefined,
  config: undefined,
  typescript: true,
  mode: 'development',
  open: undefined,
  logLevel: 'debug',
  name: 'svite',
  rollupInputOptions: { plugins: [ [Object], [Object], [Object] ] },
  transforms: [ { test: [Function: test], transform: [AsyncFunction: transform] } ],
  configureServer: [ [AsyncFunction] ],
  resolvers: [ { alias: [Function: alias] } ],
  alias: {},
  define: {},
  vueCompilerOptions: {},
  vueTransformAssetUrls: {},
  vueCustomBlockTransforms: {},
  rollupOutputOptions: undefined,
  enableRollupPluginVue: false,
  rollupDedupe: [
    'svelte/animate',
    'svelte/easing',
    'svelte/internal',
    'svelte/motion',
    'svelte/store',
    'svelte/transition',
    'svelte'
  ],
  optimizeDeps: {
    include: [
      'svelte/animate',
      'svelte/easing',
      'svelte/internal',
      'svelte/motion',
      'svelte/store',
      'svelte/transition',
      'svelte',
      'svelte-hmr/runtime/esm',
      'svelte-hmr/runtime/proxy-adapter-dom',
      'svelte-hmr/runtime/hot-api-esm'
    ],
    exclude: [ 'svelte', 'svelte-hmr' ]
  }
} +4ms
  svite: script preprocessor at index 0 is typescript. transformed: {
  code: 'const x = "y";\nexport {};\n',
  map: undefined,
  diagnostics: [],
  dependencies: []
} +190ms
[svite] overriding typescript preprocessor to support hmr in vite
{
  moduleResolution: 'node',
  target: 'es2017',
  importsNotUsedAsValues: 'error',
  isolatedModules: true,
  sourceMap: true,
  types: [ 'svelte', 'vite/dist/importMeta' ],
  strict: false,
  esModuleInterop: true,
  skipLibCheck: true,
  forceConsistentCasingInFileNames: true,
  module: 'esnext'
}
[svite] applied changes to svelte config used for dev.
{ css: true, emitCss: false }
  svite: svelte config used for dev {
  format: 'esm',
  generate: 'dom',
  css: true,
  emitCss: false,
  extensions: [ '.svelte' ],
  preprocess: [
    { script: [AsyncFunction: script] },
    { style: [AsyncFunction: style] }
  ],
  onwarn: [Function: onwarn],
  hot: {
    optimistic: true,
    noPreserveState: true,
    compatVite: true,
    absoluteImports: false
  },
  dev: true
} +1ms
  svite:  +0ms
  vite:optimize Hash is consistent. Skipping. Use --force to override. +0ms
  svite: command success +3ms
[svite]   Dev server running at:
[svite]   > Local:    http://localhost:3000/
[svite]   > Network:  http://192.168.0.208:3000/
  svite: server ready in 199ms. +1ms
  vite:hmr         / imports /src/index.ts +0ms
  vite:rewrite (skipped) / +0ms
  vite:rewrite (skipped) /vite/client +59ms
  vite:rewrite /src/index.ts: rewriting +3ms
  vite:rewrite     "./App.svelte" --> "/src/App.svelte?import" +1ms
  vite:hmr         /src/index.ts imports /src/App.svelte +65ms
  vite:rewrite     "./index.css" --> "/src/index.css?import" +1ms
  vite:hmr         /src/index.ts imports /src/index.css +0ms
  vite:hmr ws client connected +3ms
  vite:rewrite /src/App.svelte: rewriting +1s
  vite:rewrite     "svelte/internal" --> "/@modules/svelte/internal.js" +0ms
  vite:hmr         /src/App.svelte imports /@modules/svelte/internal.js +1s
  vite:rewrite     "svelte-hmr/runtime/hot-api-esm.js" --> "/@modules/svelte-hmr/runtime/hot-api-esm.js" +0ms
  vite:hmr         /src/App.svelte imports /@modules/svelte-hmr/runtime/hot-api-esm.js +0ms
  vite:rewrite     "svelte-hmr/runtime/proxy-adapter-dom.js" --> "/@modules/svelte-hmr/runtime/proxy-adapter-dom.js" +0ms
  vite:hmr         /src/App.svelte imports /@modules/svelte-hmr/runtime/proxy-adapter-dom.js +1ms
  vite:hmr rewriting /src/App.svelte for HMR. +0ms
  vite:hmr /src/App.svelte self accepts +17ms
  vite:resolve (optimized) svelte/internal.js -> node_modules/.vite_opt_cache/svelte/internal.js +0ms
  vite:resolve (optimized) svelte-hmr/runtime/hot-api-esm.js -> node_modules/.vite_opt_cache/svelte-hmr/runtime/hot-api-esm.js +1ms
  vite:resolve (optimized) svelte-hmr/runtime/proxy-adapter-dom.js -> node_modules/.vite_opt_cache/svelte-hmr/runtime/proxy-adapter-dom.js +0ms
  vite:rewrite /@modules/svelte/internal.js: no imports found. +36ms
  vite:rewrite /@modules/svelte-hmr/runtime/proxy-adapter-dom.js: no imports found. +4ms
  vite:rewrite /@modules/svelte-hmr/runtime/hot-api-esm.js: rewriting +2ms
  vite:rewrite     "../../svelte/internal.js" --> "/@modules/svelte/internal.js" +1ms
  vite:hmr         /@modules/svelte-hmr/runtime/hot-api-esm.js imports /@modules/svelte/internal.js +25ms
  vite:rewrite (skipped) /src/index.css?import +3s
^C  svite: received SIGINT, stopping server +11s

➜  borked-shell git:(master) ✗   svite: server did not stop within 3s. Exiting the hard way. +3s
```
My guess it's something with the server not stopping properly.

**Comment by dominikg** [comment]
should be fixed in 0.7.1-beta.1 . please check

**Comment by codechips** [comment]
Yep. Solved :+1:
